src/main/java/uk/gov/justice/hmpps/prison/service/BookingService.java

```diff
@@ -1078,6 +1064,7 @@ private static String forAgency(final String agencyId) {
         return AGENCY_LOCATION_ID_KEY + ":eq:'" + agencyId + "'";
     }
 
+    @VerifyOffenderAccess(overrideRoles = {"VIEW_PRISONER_DATA"})
```

Comment:
Image data NOT protected!

Reply:
Is there a potential this will break any clients who don't have this role?

Reply:
No as only used by users in prison and it will allow access if they have access to the caseload.